### PR TITLE
Fixed Xeno Pings Being Generally Visible

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Xeno pings are now visible by xenos only.

## Why It's Good For The Game

Xeno pings are now visible by xenos only.

## Changelog
:cl:
fix: Xeno pings are now visible by xenos only.
/:cl:

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -76,7 +76,7 @@
 	/// A luminescence-shifted value of the last color calculated for chatmessage overlays
 	var/chat_color_darkened
 	///HUD images that this mob can provide.
-	var/list/hud_possible
+	var/list/hud_possible = list(XENO_TACTICAL_HUD)
```
Comment (SplinterGP) [inline code/game/atoms.dm:79]: why is this on atom

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/atoms.dm (hunk 1 above)
Comment (Surrealaser) [reply code/game/atoms.dm:79]: Future proofing; I'm changing Rally Hive to atom targeting per Tivi's request

Comment (TiviPlus) [reply code/game/atoms.dm:79]: this adds a new list to **every single atom in the world**

Comment (SplinterGP) [reply code/game/atoms.dm:79]: ^

Comment (Surrealaser) [reply code/game/atoms.dm]: This should be ready to go.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**